src/components/Select/Select.scss

```diff
@@ -82,6 +82,7 @@ $stacking-order: (
   flex: 1;
   white-space: nowrap;
   overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
Cannot use `Truncate` because it adds an extra note and breaks the flexbox layout.